### PR TITLE
Fastly module does not support redirections between websites.

### PR DESCRIPTION
**Pull request by kavingas**
## Purpose of this pull request
Update the fastly GeoIP implementation 
Adobe Commerce support ticket reference https://magento.zendesk.com/agent/tickets/535551

## Affected DevDocs pages
- https://devdocs.magento.com/cloud/cdn/configure-fastly-customize-cache.html#configure-geoip-handling

### CODE REVIEW

File: src/cloud/cdn/configure-fastly-customize-cache.md [hunk 1]
```diff
@@ -98,6 +98,9 @@ To set up GeoIp handling:
 
 1. After page reload, click **Upload VCL to Fastly** in the *Fastly Configuration* section.
 
+{:.bs-callout-info}
+The current Adobe Commerce Fastly GeoIP module implementation does not support redirections between websites.
```
Comment (hguthrie) [inline src/cloud/cdn/configure-fastly-customize-cache.md:102]: Does this need to specify what kind of websites? Would multiple make sense?

```suggestion
The current Adobe Commerce Fastly GeoIP module implementation does not support redirections between multiple websites.
```

**Review by hguthrie — COMMENTED** [review]
I have a question about this note...just to clarify.

File: src/cloud/cdn/configure-fastly-customize-cache.md (hunk 1 above)
Comment (kavingas) [reply src/cloud/cdn/configure-fastly-customize-cache.md:102]: yes it make sense

**Review by hguthrie — APPROVED** [review]

**Comment by m2-github-services** [comment]
@hguthrie the branch with the code is successfully imported into the `magento-devdocs/devdocs` repository. Branch name: `imported-kavingas-devdocs-9489`.

**Pull request merged**